ynr/apps/candidates/models/popolo_extra.py

```diff
@@ -33,13 +33,18 @@ class UnsafeToDelete(Exception):
 
 
 def raise_if_unsafe_to_delete(model):
-    if model_has_related_objects(model):
+    related_models = model_has_related_objects(model)
+    if related_models:
         msg = (
             "Trying to delete a {model} (pk={pk}) that other "
-            "objects that depend on"
+            "objects that depend on ({related_models})"
```

Comment:
I know it was like this before, but there's an extra "that" in this error message

**that** other objects **that** depend on

While you're adding additional info , can we fixup the wording too :)

Reply:
Ha! Sure, pushed an amended commit